.gitignore

```diff
@@ -111,16 +111,22 @@ installer/k8s/oidc.yaml
 installer/minio/minio_ing.yaml
 installer/validate/__pycache__/
 installer/utils/dns_utils/.venvaws
+installer/utils/route53/route_record.json
+installer/helm-temp-dir
+installer/hooks/dns_entry_hook.sh
+installer/*.pem
```

Comment:
maybe we should ignore all `.crt`, `.pem` `.key` and `.csr` and not specify directories?
```
*.pem
*.crt
*.key
*.csr
```